### DISCUSSION TITLE
/zh-cn/blog/friends/

### DISCUSSION THREAD

**Discussion by Reverier-Xu** [Reverier-Xu/discussions#117] [Announcements]
https://blog.woooo.tech/friends/?code=e135e8ddb085fe2abbed

**Comment by Jacka1ope** [comment]
何德何能和上面这些巨佬并排啊QAQ

**Comment by Reverier-Xu** [comment]
> @C3ru1eaN
> 何德何能和上面这些巨佬并排啊QAQ

快去更博客（XD

**Comment by snowolf46** [comment]
突然才发现你搬家了

**Comment by Reverier-Xu** [comment]
> @snowolf46
> 突然才发现你搬家了

应该是去年换的

**Comment by arttnba3** [comment]
👴来膜全能の神RX大哥了

**Comment by Reverier-Xu** [comment]
> @arttnba3
> 👴来膜全能の神RX大哥了

❓

**Comment by dmlgzs** [comment]
膜拜 RX 神

**Comment by 66RING** [comment]
膜拜 RX 神 or2

**Comment by arttnba3** [comment]
踩踩

**Reply by Reverier-Xu** [reply]
你好强

**Reply by arttnba3** [reply]
你最强

**Reply by Reverier-Xu** [reply]
你更强

**Comment by xeonds** [comment]
膜rx神.gif

**Comment by n-WN** [comment]
想要交换友链：https://lov2.netlify.app/

你的博客我经常作为消遣看，夸夸博客里的图片，拍的挺好看的，糖葫芦很诱人

**Comment by YunZh1Jun** [comment]
何德何能，竟然能出现在rx神的友链里，此生无憾了😭😭😭

**Reply by Reverier-Xu** [reply]
别别别，能放🦘国女总理的友链是我荣幸

**Comment by BenderBlog** [comment]
Query for my page: http://legacy.superbart.top. Thank you.

**Reply by Reverier-Xu** [reply]
why legacy

**Reply by Reverier-Xu** [reply]
done.
